src/less/availability.less

```diff
@@ -1,19 +1,11 @@
 @import './variables.less';
 
-.rbc-availability,
-.rbc-availability-overlap {
+.rbc-availability {
   background-color: @availability-bg;
   border: @availability-border;
   border-radius: @availability-border-radius;
   color: @availability-color;
-}
-
-.rbc-availability {
+  cursor: pointer;
```

Comment:
This seems like something that shouldn't be applied for all availabilities unless all availabilities are guaranteed to have an onClick/onSelect handler. Is that the case? Because if not, we'll be making availabilities look clickable when they aren't. A better approach in that case would be to only set this style for availabilities that do have onClick/onSelects.

My understanding is that it's **not** true that all availabilities are guaranteed to have click/select handlers, that will only be true when the `onSelectAvailability` or whatever prop is passed into the Calendar.

Reply:
Good catch, I'll add a guard.